README.md

```diff
@@ -1,29 +1,28 @@
-# morello-api
+# DSBD-API 
```

Comment:
Should Match repository name, if necessary we can rename the repository.

Reply:
changed already